corona-warn-app/cwa-app-android
# Increase tap area of infobuttons (EXPOSUREAPP-11080)

Jira: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11080

![grafik](https://user-images.githubusercontent.com/10398034/146781713-0de73c14-87a6-4edb-b79b-06d6b74efe14.png)

How to test
- Scan certificate
- Check if tap areas of info button is large enough in overview screen and details screen
